packages/spatial/tests/webxr/emulator.ts

```diff
@@ -57,14 +58,30 @@ export function getLastXRSessionData(session: XRSession | null) {
   return Array.from(getXRSessionData(session).device.sessions).at(-1).at(-1)
 }
 
+/**
+ * @warning
+ * Cannot work until this symbol from webxr-polyfill is exported
+ * ```ts
+ * const PRIVATE = Symbol('@@webxr-polyfill/EventTarget');
+ * ```
+ * Link: [webxr-polyfill/EventTarget.js#L16](https://github.com/immersive-web/webxr-polyfill/blob/main/src/lib/EventTarget.js#L16)
+ *
+ * @description Returns the EventTarget data of the `@param session` XRSession passed in by accessing it with its Symbol() name
+ * @why Shorthand for getting the EventTarget data of the session in an ergonomic way.
+ * */
+export function __getXRSessionEventTargetData(session: XRSession | null) {
+  if (session === null) return null
+  return session[EVENTTARGET_SYMBOL]
+}
+
 /**
  * @description Requests an emulated XRSession.
  * @why Shorthand for initializing an emulated XRSession from unit tests
  * */
 export async function requestEmulatedXRSession(deviceDefinition = DeviceDefinitions.Default) {
   WebXREventDispatcher.instance.dispatchEvent({
     type: POLYFILL_ACTIONS.DEVICE_INIT,
-    detail: { stereoEffect: false, deviceDefinition }
+    detail: { stereoEffect: false, device: deviceDefinition }
```

Comment:
`device` should be `deviceDefinition`

Reply:
Fixed with 81bce7e0aa87a77edee4ea2238475a826b52661e